htdocs/takepos/ajax/ajax.php

```diff
@@ -283,6 +289,10 @@
 		$sql .= $hookmanager->resPrint;
 	}
 
+	if (getDolGlobalInt('TAKEPOS_PRODUCT_IN_STOCK') == 1 && empty(getDolGlobalInt('CASHDESK_ID_WAREHOUSE'.$_SESSION['takeposterminal']))) {
+		$sql .= ' GROUP BY p.rowid HAVING SUM(ps.reel) > 0';
```

Comment:
The select contai s rowid but also other fields.
To be compliant with sql standard and all daya ase setup, a group by must always contains all the fields in the select except the sum of course...

Reply:
hello @eldy, i have fixed the group by.